StyleCop.Analyzers/StyleCop.Analyzers/ReadabilityRules/SA1133DoNotCombineAttributes.cs

```diff
@@ -32,7 +38,22 @@ internal class SA1133DoNotCombineAttributes : DiagnosticAnalyzer
         /// <inheritdoc/>
         public override void Initialize(AnalysisContext context)
         {
-            // TODO: Implement analysis
+            context.RegisterCompilationStartAction(CompilationStartAction);
+        }
+
+        private static void HandleCompilationStart(CompilationStartAnalysisContext context)
+        {
+            context.RegisterSyntaxNodeActionHonorExclusions(HandleAttributeListAction, SyntaxKind.AttributeList);
+        }
+
+        private static void HandleAttributeList(SyntaxNodeAnalysisContext context)
+        {
+            AttributeListSyntax attributeList = (AttributeListSyntax)context.Node;
+
+            if (attributeList.Attributes.Count > 1)
+            {
+                context.ReportDiagnostic(Diagnostic.Create(Descriptor, attributeList.Attributes[1].GetLocation()));
```

Comment:
:bulb: I believe this highlights the entire argument list for the attribute if one is present. I would prefer to see it only mark the (possibly qualified) attribute type name.